actions/service.go

```diff
@@ -421,3 +423,8 @@ func (s *Service) PostMergeHook(ctx context.Context, record graveler.HookRecord)
 	// post merge actions will be enabled here
 	return nil
 }
+
+func NewHookRunID(id int) string {
+	tm := time.Now().UTC().Format("02150405")
+	return fmt.Sprintf("H%s%06d", tm, id)
```

Comment:
What does the `H` stand for?

Reply:
Will remove it - used it first to identify the hook run id